### ISSUE TITLE
Would it be nice to bundle the upstream CoreDNS version into the k8s-gateway release?

### ISSUE THREAD

**Issue by mstoetzer** [ori-edge/k8s_gateway#98]
Something like <CoreDNS_UPSTREAM>-<Plugin_VERSION> e.g. `image: quay.io/oriedge/k8s_gateway:1.9.1-0.3.0`

**Comment by networkop** [comment]
yeah, I think this is possible. I've never seen this pattern before though. Any other plugins that implement this?

**Comment by mstoetzer** [comment]
My idea was to use that version in the image tag. I just wanted to trigger some discussion about that :)

**Comment by networkop** [comment]
I think it makes sense. Obv, we won't be building the same plugin version for different upstream coredns versions, but we can add an extra tag to an existing image for extra visibility.

**Comment by Ornias1993** [comment]
Double version is a hell to work with when you update containers using tools like renovate.
It works and can be made to work, but needless double versioning is usually more than a chore than a benefid imho.
